library/src/noop/java/com/telefonica/tweaks/Tweaks.kt

```diff
@@ -18,6 +18,15 @@ open class Tweaks {
         return getTweakValue(entry)
     }
 
+    open fun <T> getTweakValue(key: String, defaultValue: T): Flow<T> =
+        flowOf(defaultValue)
+
+    open suspend fun <T> getTweak(key: String): T? =
+        null
```

Comment:
I think something is missing here 😄

Reply:
This is the no-op module. We use it to provide an implementation of Tweaks that don't do anything, so we provide it to the `release` build, while the real one only goes to `debug`: https://github.com/Telefonica/b2p-germany-android/blob/main/app/build.gradle#L250

You can check it correctly in the `flavors` view

Reply:
Oops, sorry, didn't know that. I will see it.

Reply:
I have added a better implementation for the `no-op` scenario, where we try to find a default value provided in the app, instead returning the default one or null